addon/components/bgr-breadcrumbs.js

```diff
@@ -1,39 +1,39 @@
-import Component from '@ember/component';
-import { inject as service } from '@ember/service';
-import layout from '../templates/components/bgr-breadcrumbs';
+import Component from '@ember/component'
+import { inject as service } from '@ember/service'
+import layout from '../templates/components/bgr-breadcrumbs'
 
 export default Component.extend({
   /**
-   * injections
+   * Services
    */
 
-  bgrBreadcrumbs: service(),
+  breadcrumbsService: service('bgr-breadcrumbs'),
 
   /**
-   * props
+   * Arguments
    */
 
   className: 'breadcrumbs',
   itemClassName: 'breadcrumbs__item',
   linkClassName: 'breadcrumbs__link',
 
   /**
-   * state
+   * State
    */
 
   classNameBindings: ['className'],
   layout,
   tagName: 'ul',
 
   /**
-   * hooks
+   * Hooks
    */
 
-  didInsertElement() {
-    this.get('bgrBreadcrumbs.instances').addObject(this);
+  didInsertElement () {
+    this.get('breadcrumbsService.instances').addObject(this)
```

Comment:
Unrelated to this PR, but I would add helper methods to the breadcrumbs service. addInstance / removeInstance or registerInstance/unregisterInstance. This component doesn't have to know that this is implemented as an EmberArray.